blueoil/utils/predict_output/output.py

```diff
@@ -118,18 +118,17 @@ def _semantic_segmentation(self, outputs, raw_images, image_files):
             }
 
             prediction = []
+            label = np.argmax(output, axis=2)
             for i, class_name in enumerate(self.classes):
-                img = output[:, :, i] * 255
-                img = PIL.Image.fromarray(img, mode="F")
-                img = img.convert("L")
+                img = np.zeros(label.shape[:2], dtype=np.uint8)
+                img[label == i] = 255
+                img = PIL.Image.fromarray(img, mode="L").convert('1')
```

Comment:
@oatawa1 
Sorry, I don't understand why this diff solve the problem
Can you teach me?

Reply:
I committed again to make it similar to the previous commit.

The new commit assigns each pixel color by the predicted class (the highest probability from softmax), Thus, it can be seen as assign 255 to the pixel that belongs to that class.
$ \sum_{\for i=class_no}{max(p_i) * 255} $

While each pixel in the old commit is a combination of each class based on softmax probability.
$ \sum_{\for i=class_no}{p_i * 255} and sum(p_i) = 1.0 $